__tests__/api/customers.test.js

```diff
@@ -0,0 +1,50 @@
+const {
+    getPool,
+    createTables, 
+    createCustomer, 
+} = require('../../server/db');
+
+const {
+    getBilbo,
+    getDrogo,
+    getBozo
+} = require('../../server/db/utils');
+
+const {app} = require('../../server')
+const supertest = require('supertest');
+const request = supertest(app)
+
+const pool = getPool();
+
+describe( 'api/customers', () => {
+    afterAll(async () => {
+        await pool.end()
+    })
+
+describe('GET api/customers', () => {
+
```

Comment:
What is the purpose of the nested Describe here?

```suggestion
describe( 'api/customers', () => {
    afterAll(async () => {
        await pool.end()
    })
```

Reply:
Nevermind, I see that the other PR is adding POST tests in the same file, makes sense to me